eZ/Publish/Core/Persistence/Legacy/Content/Gateway.php

```diff
@@ -459,4 +459,12 @@ abstract public function deleteTranslationFromVersion(
         int $versionNo,
         string $languageCode
     ): void;
+
+    /**
+     * @param array<int> $contentIds
+     *
+     * @throws \Doctrine\DBAL\Driver\Exception
+     * @throws \Doctrine\DBAL\Exception
```

Comment:
This should never be thrown by the ExceptionConversion layer, which is the last one aliasing this abstract. It's there for that reason. You can report here `\eZ\Publish\Core\Base\Exceptions\DatabaseException` thrown. If it indeed throws at least one of the mentioned exceptions, ExceptionConversion layer should take care about that.